content/releases/cloudnative-pg-1-26.0-rc1-released/index.md

```diff
@@ -0,0 +1,164 @@
+---
+title: "CloudNativePG 1.26.0 RC1 Released!"
+date: 2025-03-28T11:03:23+01:00
+draft: false
+author: gbartolini
+image:
+    url: 58364dde1adc4a0a8.37186404-2048x1445.jpg
+    attribution: from <a href="https://wordpress.org/photos/photo/58364dde1a/">Saurabh</a>
+tags:
+ - release
+ - postgresql
+ - postgres
+ - kubernetes
+ - k8s
+ - cloudnativepg
+ - preview
+ - cnpg
+summary: "The CloudNativePG community is excited to announce the first release candidate of CloudNativePG 1.26! This preview introduces features like declarative offline in-place major upgrades of PostgreSQL, startup and readiness probes for replicas, declarative management of extensions and schemas, and more. Join us in testing these updates to shape the final release."
+---
+
+
+The **CloudNativePG Community** is thrilled to announce the first release
+candidate of CloudNativePG 1.26! This preview release provides an opportunity
+to explore new features and enhancements before the final version is officially
+launched. While refinements may still occur, here’s a look at what’s new.
+
+## Key Features
+
+### Declarative Offline In-Place Major Upgrades of PostgreSQL
+
+You can now trigger an offline in-place major upgrade by specifying a new
+operand container image with a higher PostgreSQL major version in a cluster,
+either directly or via image catalogs. During the upgrade, the cluster is shut
+down to ensure data consistency, and a `pg_upgrade` job performs the migration.
+This long-awaited feature simplifies major upgrades while maintaining
+reliability.
+
+### Enhanced Startup and Readiness Probes for Replicas
+
+We have improved Kubernetes startup and readiness probes for PostgreSQL
+instances, introducing a unique capability: the readiness of a replica can now
+be controlled based on its lag from the primary. For example, this allows you
```

Comment:
Remove 'for example'

Reply:
The idea here is to say that this is just one of the possible use cases.

Reply:
hm ok